README.md

```diff
@@ -178,17 +179,30 @@ If you try to sign-in before the tenant administrator has provisioned the app in
 ![AADSTS700016, App not found](./ReadmeFiles/AADSTS700016.JPG)
 
 ## National Cloud Deviations
-
-In order to run this sample on a National Cloud, you can follow through the steps above with a few variations:
-
-1. You must register this sample for your Azure AD Tenant in a [National Clouds](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud) by following [[Step 2]](#step-2-register-the-sample-with-your-azure-active-directory-tenant) above in the National cloud of your choice.
- If you using automation provided via Powershell to create your app, you need to change the [Configure.ps1](./AppCreationScripts/Configure.ps1) and [Cleanup.ps1](./AppCreationScripts/Cleanup.ps1) as instructed below to append the `-AzureEnvironmentName` parameter. The details on this parameter and its possible values are listed in [Connect-AzureAD](https://docs.microsoft.com/en-us/powershell/module/azuread/connect-azuread?view=azureadps-2.0).
-
- ```Powershell
- Connect-AzureAD -TenantId $tenantId -AzureEnvironmentName AzureUSGovernment
- ```
-
-1. Then follow the steps outlined in [[Steps 3]](#step-3--configure-the-sample-to-use-your-azure-ad-tenant) above and additionally make the following changes in the `TodoListWebApp\Web.Config` file.
+Microsoft cloud services are also available in three separate national clouds. These national cloud versions are physical and logical network-isolated instances of Microsoft enterprise cloud services that are confined within the geographic borders of specific countries and operated by local personnel.
+Current national clouds include:
+- Microsoft Cloud for US Government
+- Microsoft Cloud Germany
+- Azure and Office 365 operated by 21Vianet in China
+
+In order to run this sample on a National Cloud, you can follow the same steps as of Global cloud with a few variations:
+
+#### Step 1: 
+Follow [[Step 1]](#Clone-or-download-this-repository)
+
+#### Step 2: 
+1. Sign in to the Azure Government portal https://portal.azure.us.
```

Comment:
You particularized the Azure gov here? I wonder if we would not want to have a sub section directly in step 2. Something like:

> If you want to create an app for Microsoft national or sovereign clouds, sign-in to the portal corresponding to that cloud:
  - https://portal.azure.us for the Azure Government
 - See [link to the sovereign cloud table]() for others